### DISCUSSION TITLE
In Avalonia 11P6 -- NumericUpDown.Value ??

### DISCUSSION THREAD

**Discussion by kuiperzone** [AvaloniaUI/Avalonia#10755] [General]
In the 11P6, NumericUpDown has:

public decimal? Value { get; set; }

I note the change to decimal?

But more importantly, what does a null value indicate and how should I handle it?

**Comment by timunie** [comment]
You can just bind to a decimal? in your ViewModel if you want to. That way you can have a blank input field if needed. As an alternative you can check in the setter for `null` and set "0" instead. 

Usecases:
- You can show a watermark in a blank field 
- If the field is mandatory, but 0 is a valid number, you can indicate to the user that input should be made. This redurces potential errors by users overseeing one of many input fields

More about it here: https://github.com/AvaloniaUI/Avalonia/issues/7725

**Reply by kuiperzone** [reply]
Thank you.

If I understand you correctly, a null Value simply means that the control is empty, i.e. not "0", but just ""?

**Reply by timunie** [reply]
Yes, exactly.
